src/timer.rs

```diff
@@ -78,18 +76,25 @@
 //!
 //! [runtime]: ../runtime/struct.Runtime.html
 //! [tokio-timer]: https://docs.rs/tokio-timer
-//! [ext]: ../util/trait.FutureExt.html#method.deadline
-//! [Deadline]: struct.Deadline.html
+//! [ext]: ../util/trait.FutureExt.html#method.timeout
+//! [Timeout]: struct.Timeout.html
 //! [Delay]: struct.Delay.html
 //! [Interval]: struct.Interval.html
 //! [`DelayQueue`]: struct.DelayQueue.html
 
 pub use tokio_timer::{
     delay_queue,
-    Deadline,
-    DeadlineError,
     DelayQueue,
     Error,
     Interval,
     Delay,
+    Timeout,
+    timeout,
+};
+
+#[deprecated(since = "0.1.8", note = "use Timeout instead")]
+#[doc(hidden)]
+pub use tokio_timer::{
```

Comment:
Should these deprecations just go on the `struct Deadline` in the original module?

Reply:
Probably... not sure why I didn't originally do that.

Reply:
So, thinking about it more, it seems useful to explicitly deprecate separately in the main crate vs. the sub crates because a) the version at which they were deprecated is different and b) the re-export might be deprecated but not the actual type in the sub crate (like the current thread executor).